.azure-pipelines/buildAndPackage.yml

```diff
@@ -0,0 +1,96 @@
+trigger:
```

Comment:
Add copyright header and comments about the intended purpose of this pipeline definition.